drivers/aip31068/include/aip31068_regs.h

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (C) 2020 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_aip31068
+ * @brief       Register definitions for the AIP31068 I2C LCD controller
+ * @author      Hendrik van Essen <hendrik.ve@fu-berlin.de>
+ * @file
+ * @{
+ */
+
+#ifndef AIP31068_REGS_H
+#define AIP31068_REGS_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/* Commands (page 18, table 3) */
+
+/**
+ * @brief Clears entire display and sets cursor to position (0, 0).
+ *
+ * ATTENTION: Also changes BIT_ENTRY_MODE_INCREMENT to 1
+ */
+#define CMD_CLEAR_DISPLAY   0x01
```

Comment:
Not insisting on this but you could aline all the values.

Reply:
I will align the values within their groups.